ceph-dev-setup/build/build

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash -ex
+# -*- mode:sh; tab-width:4; sh-basic-offset:4; indent-tabs-mode:nil -*-
+# vim: softtabstop=4 shiftwidth=4 expandtab
```

Comment:
I love Vim, but do we really need to introduce editor-specific configuration in files? This can all be configured locally. Not everyone uses Vim either

Reply:
i agree that not everyone uses VIM. and the previous line is for Emacs users =)

and yeah, i will remove them. the reason i added the editor variables was that i want to avoid mix of tab and spaces in future -- like what we have in `vstart.sh` now.

Reply:
@alfredodeza updated and repushed